### PR TITLE
Crashlytics issues upload to Redmine

### PR DESCRIPTION
**Pull request by BenoitDuffez**
Fabric Crashlytics used to have a service hook to push crash data to Redmine.

As discussed with Mike from Firebase/Fabric (sic), Firebase functions can do that. As per the request on [Stack Overflow](https://stackoverflow.com/questions/52192755/how-can-we-continue-to-use-the-redmine-service-hook-with-the-new-firebase-crashl#comment91477939_52195523), here is my contribution about that.

### CODE REVIEW

File: redmine-issue/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+```
+firebase deploy --only functions
+```
+
+## What does it do?
+
+Once you have configured the API key (please create a dedicated user for that), server URL, project identifier (numeric or string) and the issue type (tracker, by default 1=Bug):
+
+1. The code is triggered by an issue on Firebase Crashlytics
```
Comment (nicolasgarnier) [inline redmine-issue/README.md]: The code is triggered when a new issue is created on Firebase Crashlytics

File: redmine-issue/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+1. The list of available issue priorities is downloaded from Redmine
+1. The best issue priority is found, based on Firebase Crashlytics issue settings and Redmine default issue priority
+1. The issue is then created on Redmine
+
+# Contribute
+
+I've never done any Firebase Functions development so I have no idea if this is how things should be done. Any contribution welcome.
```
Comment (nicolasgarnier) [inline redmine-issue/README.md]: Replace with "We'd love that you contribute to the project. Before doing so please read our [Contributor guide](../CONTRIBUTING.md)."

**Comment by BenoitDuffez** [comment]
Will do, thanks for the review.

**Comment by BenoitDuffez** [comment]
@nicolasgarnier: I *finally* took the time to take your review into account, kindly have a look at the PR and see if it can still be merged (it's been a while).
Thanks!

**Comment by jhuleatt** [comment]
[cleaning up old issues]

Hey @BenoitDuffez, sorry we never merged your PR! If you're still interested in this, feel free to open a new PR against the `master` branch

**Pull request closed without merging**